addons/web_editor/static/src/js/editor/odoo-editor/test/spec/powerbox.test.js

```diff
@@ -1,13 +1,31 @@
 import { setSelection } from '../../src/OdooEditor.js';
 import { Powerbox } from '../../src/powerbox/Powerbox.js';
-import { BasicEditor, _isMobile, insertText, testEditor, triggerEvent } from '../utils.js';
+import { BasicEditor, _isMobile, insertText, nextTick, testEditor, triggerEvent } from '../utils.js';
 
 const getCurrentCommandNames = powerbox => {
     return [...powerbox.el.querySelectorAll('.oe-powerbox-commandName')].map(c => c.innerText);
 }
 
 describe('Powerbox', () => {
     describe('integration', () => {
+        it('should not reinsert the selected text after command validation', async () => {
+            await testEditor(BasicEditor, {
+                contentBefore: '<p>[]<br></p>',
+                stepFunction: async editor => {
+                    await insertText(editor, 'abc');
+                    const p = editor.editable.querySelector('p');
+                    setSelection(p.childNodes[0], 0, p.childNodes[2], 1);
+                    await nextTick();
+                    window.chai.expect(editor.powerbox.isOpen).to.eql(false);
+                    window.chai.expect(editor.powerbox.el.style.display).to.eql('none');
```

Comment:
I don't see the need for this check when you have already asserted `powebox.isOpen`.